src/components/AddEditMovie/AddEditMovie.jsx

```diff
@@ -1,130 +1,175 @@
 /* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
+import { Formik } from 'formik';
+
 import {
-  Card,
-  Form,
+  SubmitButton,
   Input,
-  Button,
-  DatePicker,
-  Select,
   Checkbox,
-  Row
-} from 'antd';
+  ResetButton,
+  Form,
+  Select,
+  DatePicker,
+  FormItem
+} from 'formik-antd';
+
+import { Card, Button, Row, message, Col } from 'antd';
 
 import { genres } from '../../mock_data';
 
 import { useModal } from '../../hooks';
 
 import 'antd/dist/antd.css';
+import './AddEditMovie.modules.scss';
 
 const { Option } = Select;
 
-const formItemLayout = {
-  labelCol: {
-    xs: {
-      span: 24
-    },
-    sm: {
-      span: 8
-    }
-  },
-  wrapperCol: {
-    xs: {
-      span: 24
-    },
-    sm: {
-      span: 16
-    }
-  }
-};
-const tailFormItemLayout = {
-  wrapperCol: {
-    xs: {
-      span: 24,
-      offset: 0
-    },
-    sm: {
-      span: 16,
-      offset: 8
-    }
-  }
+const validateRequired = (value) => {
+  return value ? undefined : 'required';
 };
 
 export const AddEditMovie = () => {
   const { state } = useModal();
+  const {
+    editContent: { description, image, movieType, name, rating, runtime },
+    id
+  } = state;
   const { openAdd, openEdit, title } = state;
-  const [form] = Form.useForm();
-  const onFinish = (values) => {
-    console.log('Received values of form: ', values);
-  };
 
   const cardWithAddEdit = openAdd || openEdit;
 
+  const deleteMove = () => {
```

Comment:
I have to implemented this functionality:
![image](https://user-images.githubusercontent.com/52933810/161117068-49d5765a-acf2-48c9-8dc9-b99dfb5f0c6c.png)

I think to be able to delete movies you should keep movies in a state instead of mock data, smth like that:

![image](https://user-images.githubusercontent.com/52933810/161120238-2c8699b2-b809-49b6-9da1-d893a0952746.png)